java/test/src/main/java/test/Ice/packagemd/NoPackage.ice

```diff
@@ -25,7 +25,7 @@ exception E2 extends E1
     long l;
 }
 
-exception notify /* Test keyword escape. */
```

Comment:
`notify` is not a keyword, but it is a 'special' identifier, defined on every Object.
Here, there _should_ be no problem with it as an exception name though.

Reply:
What are we testing here? If a test is redundant, it should be deleted.

Reply:
It's like I said, it seems like we're testing that you can throw and catch exceptions that have been re-mapped with `java:package`. If you're asking for the specific code snippet, here:
```
            try {
                initial.throwTest1Notify();
                test(false);
            } catch (notify ex) {
                // Expected
            }
```
`throwTest1Notify` throws a `notify` exception, and then we catch it.

After this we have tests that seem to be testing slicing in this context as well, with these 2 other operations:
```
    void throwTest1E2AsE1();
    void throwTest1E2AsE2();
```
But these seem like more advanced cases, whereas `notify` is testing a 'base case'.